rerun_py/tests/unit/test_time_range_boundary.py

```diff
@@ -7,47 +7,47 @@
 def test_time_range_boundary_failure_cases() -> None:
     # Too many arguments for absolute.
     with pytest.raises(ValueError):
-        rr.TimeRangeBoundary.absolute(rr.TimeInt(seq=0), seq=123)
+        rr.TimeRangeBoundary.absolute(rr.TimeInt(seq=0), seq=123)  # type: ignore[call-overload]
```

Comment:
Can the `type: ignore[call-overload]` be applied to the entire contents of the function or file instead, to reduce the visual clutter?

Reply:
i didn't find a way for that, and tbh i like visibility for that stuff, as its there for a reason (even if it looks ugly)